src/containers/Tenant/Query/QueryResult/components/QueryInfoDropdown/QueryInfoDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import {Ellipsis} from '@gravity-ui/icons';
+import type {ButtonProps} from '@gravity-ui/uikit';
+import {Button, DropdownMenu} from '@gravity-ui/uikit';
+
+import type {QueryResultsInfo} from './useQueryInfoMenuItems';
+import {useQueryInfoMenuItems} from './useQueryInfoMenuItems';
+
+import './QueryInfoDropdown.scss';
+
+interface QueryInfoDropdownProps {
+    queryResultsInfo: QueryResultsInfo;
+    database: string;
+    hasPlanToSvg: boolean;
+}
+
+export function QueryInfoDropdown({
+    queryResultsInfo,
+    database,
+    hasPlanToSvg,
+}: QueryInfoDropdownProps) {
+    const {isLoading, items} = useQueryInfoMenuItems({
+        queryResultsInfo,
+        database,
+        hasPlanToSvg,
+    });
+
+    const renderSwitcher = (props: ButtonProps) => {
+        return (
+            <Button view="flat-secondary" loading={isLoading} disabled={isLoading} {...props}>
```

Comment:
it seems wrong title:
<img width="160" alt="Screenshot 2025-01-21 at 16 20 03" src="https://github.com/user-attachments/assets/e10d066d-bf0b-4b9e-bf5c-9fa0932087cb" />

Btw let's use `ActionTooltip` for titles.

Reply:
<img width="276" alt="image" src="https://github.com/user-attachments/assets/ebd61aac-0084-4ac4-b8fb-68977942f33e" />
it is not